docs/_static/nuts-registry.yaml

```diff
@@ -248,6 +248,10 @@ components:
           example: Medicare Software for People B.V.
         identifier:
           $ref: "#/components/schemas/Identifier"
+        domain:
```

Comment:
Let's define an enum with at least: healthcare, personal and insurance

(personal being the general term for pgo)

Reply:
and make it a model

Reply:
shouldn't it be a mandatory param or default to 'healthcare' (backwards compatible)?

Reply:
Fixed